test/mocha/10-asymmetric-key.js

```diff
@@ -68,6 +95,138 @@ describe('asymmetric keys', () => {
+          err.name.should.equal('Error');
+        });
+    });
+
+    describe('unwrapKey API', () => {
+      it('throws an Error when trying to unwrap a key',
+        async () => {
+          const wrappedKey = '8vEgpnq8F6QVRmaSYPHTKKZyCXMOgRLiBdZPcfYnIfI';
+
+          let result;
+          let err;
+          try {
+            result = await brSSM.unwrapKey(
+              {keyId, operation: {wrappedKey}});
+          } catch(e) {
+            err = e;
+          }
+
+          should.exist(err);
+          should.not.exist(result);
+          err.name.should.equal('Error');
+        });
+    });
+
+    describe('verify API', () => {
+      it('throws an Error when trying to verify',
+        async () => {
+          const plaintextBuffer = Buffer.from(uuid(), 'utf8');
+          const verifyData = base64url.encode(plaintextBuffer);
+          const operation = {
+            verifyData
+          };
+          const result = await brSSM.sign({keyId, operation});
+          const {signatureValue} = result;
+
+          let result2;
+          let err;
+          try {
+            result2 = await brSSM.verify(
+              {keyId, operation: {signatureValue, verifyData}});
+          } catch(e) {
+            err = e;
+          }
+
+          should.exist(err);
+          should.not.exist(result2);
+          err.name.should.equal('Error');
+        });
+    });
+
+    describe('deriveSecret API', () => {
+      it('throws an Error when trying to derive a secret', async () => {
+        const keyId = `https://example.com/kms/${await generateId()}`;
+        const controller = 'https://example.com/i/foo';
+        const type = 'Ed25519VerificationKey2018';
+        const invocationTarget = {id: keyId, type, controller};
+        await brSSM.generateKey(
+          {keyId, operation: {invocationTarget}});
+
+        let result;
+        let err;
+        try {
+          result = await brSSM.deriveSecret(
+            {keyId, operation: {invocationTarget}});
+        } catch(e) {
+          err = e;
+        }
+
+        should.exist(err);
+        should.not.exist(result);
+        err.name.should.equal('Error');
+      });
+      it('throws an Error if public and key type do not match inside ' +
+      '_keyAgreementKey.deriveSecret() helper function', async () => {
+        const key = {
+          type: 'typeA'
+        };
+        const publicKey = {
+          type: 'typeB'
+        };
+
+        let result;
+        let err;
+        try {
+          result = await brSSM._helpers._keyAgreementKey.deriveSecret(
+            {key, operation: {publicKey}});
+        } catch(e) {
+          err = e;
+        }
+
+        should.exist(err);
+        should.not.exist(result);
+        err.name.should.equal('Error');
+      });
+    });
+
+    describe('generateKeyPair API', () => {
+      it('throws an Error if invalid type is used inside ' +
+      '_keyAgreementKey.generateKeyPair() helper function', async () => {
```

Comment:
Is linting running on the test suite here?

```suggestion
        '_keyAgreementKey.generateKeyPair() helper function', async () => {
```